frontend/src/components/ElectionForm/ElectionDetails.tsx

```diff
@@ -127,28 +125,48 @@ export default function ElectionDetails({ election, applyElectionUpdate, getStyl
                 <FormHelperText error sx={{ pl: 1, pt: 0 }}>
                     {errors.description}
                 </FormHelperText>
+            </Grid>            
+            <Grid item xs={4} sx={{ m: 0, p: 1 }} justifyContent='center'>
+                <FormControl fullWidth>
+                    <InputLabel id="demo-simple-select-label">Time Zone</InputLabel>
+                    <Select
+                        labelId="demo-simple-select-label"
+                        id="demo-simple-select"
+                        value={timeZone}
+                        label="Time Zone"
+                        onChange={(e) => setTimeZone(e.target.value)}
+                    >
+                        <MenuItem value={timeZone}>{timeZone}</MenuItem>
+                        {timeZones.map(tz =>
+                            <MenuItem value={tz.ID}>{tz.name}</MenuItem>
```

Comment:
Can we include the timezone offset in the listing? That way if someone doesn't see their location in the list, they can pick the one that matches their timezone offset

Zoom does this as well
![image](https://github.com/Equal-Vote/star-server/assets/9289903/8c403ad6-6e1c-467f-94de-2922ce3e83ae)

I could see this being complicated since the offsets change with day light savings, so I'm also ok with making an issue for now and leaving it for a future PR

Reply:
Hmm, I originally didn't do that because they change but I can see the utility of it. Lets make an issue for it for now.